tests/discovery.spec.coffee

```diff
@@ -53,6 +54,18 @@ describe 'Discoverable Services:', ->
 			if dummyService.tags?
 				expect(service.tags).to.deep.equal(dummyService.tags)
 
+	inspectPublishedAddresses = (service, hostIPs) ->
+		if hostIPs?
+			ips = hostIPs
+		else
+			interfaces = os.networkInterfaces()
+			ips = []
+			_.forEach _.flatten(_.values(interfaces)), (value) ->
+				ips.push(value.address) if value.internal == false
+
+		_.intersection(ips, service.addresses).length is ips.length
```

Comment:
This is trying to make sure that a service is published on all ips right? And it doesn't care if it's also published on additional ips?